Apps/WebClient/src/ClientApp/app/models/medicationTimelineEntry.ts

```diff
@@ -69,6 +69,7 @@ class MedicationViewModel {
 
   public populateFromModel(model: MedicationResult): void {
     //console.log(model);
+    this.isPin = !model.federalData;
```

Comment:
Since we have an IF right next this looks kind of odd. What do you think about moving it inside the if {} else {} and set the value there?

Reply:
Changed